### ISSUE TITLE
miniconda re-installation - specifications in conflict  (Libpython for python3.5)

### ISSUE THREAD

**Issue by borealis88** [conda/conda#3101]
Hello, I am having trouble getting conda to work. Would appreciate any help. Am using Window 10.

I installed Miniconda3 and want to install libpython. When entering "conda install libpython" I get the following error message: "The following specifications were found to be in conflict", followed by the following list:

[Install_libpython_error_160716.txt](url)

What I had done earlier:
1. Installed Anaconda with Python 2.7
2. Decided to upgrade to 3.5, so uninstalled Anaconda/2.7 components.
3. Installed Anaconda with Python 3.5.
4. Was unable to install libpython
5. Uninstalled Anaconda, installed Miniconda 3
6. Unable to install libpython

Why do I want install libpython? I am trying to set up with Theano as described here. http://lepisma.github.io/articles/2015/07/30/up-with-theano-and-cuda/. The instruction is to install libpython.

In other words, I am unable to follow these instructions as I cannot install libpython. But the overarching issue is that I would like to set up Theano. In fact I was able to get it to work while I was still using python 2.7 - the problem started when I wanted to use 3.5 instead.

I am a novice to this, so greatly appreciating any help.

**Comment by HugoTian** [comment]
Hope this link https://github.com/ContinuumIO/anaconda-issues/issues/561
may help

**Comment by borealis88** [comment]
Thank you @HugoTian . Will take a look!

**Comment by borealis88** [comment]
Thanks @HugoTian for your response last week. Since your link said there is no libpython for Python 3.5, I decided to go back to Python 2.7. I uninstalled Miniconda 3 from the system, then downloaded and ran the Anaconda installer for Python 2.7.  The installation completed without any issues.

A problem now occurs when I run the Jupyter Notebook and click on "New", then selecting "Python 2". In the new tab that appears, there is a "Kernel Error". When I click on the "Kernel Error" a box pops up:
![160727_kernel-error](https://cloud.githubusercontent.com/assets/19617554/17179211/907d2882-5442-11e6-9dc6-2e2f6ccf24a8.JPG)

The box contains the following information:
[160727_Kernel-error.txt](https://github.com/conda/conda/files/386326/160727_Kernel-error.txt)

What can I do to resolve this issue?

Earlier (a few weeks back), I had Python 2.7 (Anaconda) installed and it worked fine. The problem started when I uninstalled 2.7 and installed 3.5 (again Anaconda). I then ran into the same kernel issue. Subsequently I uninstalled 3.5 and tried with Miniconda and ran into the problem described above in the original post.  By the way, the Jupyter notebook still retains the option to use "Python 3" even though I have already uninstalled python 3.5...

Greatly appreciating any help with this!

**Comment by borealis88** [comment]
Further to my comment above:

The following further error messages are included in files saved at the bottom of the file and folder list in the Jupyter Notebook:

​Error! Permission denied: NTUSER.DAT
Saving disabled.
See Console for details.


Error! C:\Users\HH\NTUSER.DAT{485596c4-7ed5-11e5-80df-e41d2d718e10}.TMContainer00000000000000000001.regtrans-ms is not UTF-8 encoded
Saving disabled.
See Console for more details.

​​​​​​
​Error! C:\Users\HH\NTUSER.DAT{485596c4-7ed5-11e5-80df-e41d2d718e10}.TM.blf is not UTF-8 encoded
Saving disabled.
See Console for more details.


​Error! C:\Users\HH\ntuser.ini is not UTF-8 encoded
Saving disabled.
See Console for more details.
​​​

**Comment by kalefranz** [comment]
This is the wrong issue tracker for you issue.  There's nothing you're encountering here that changing conda's own code can help with.
